docs/sinco/v4.x/2_tutorials/3_page/10_wait_for_requests.md

```diff
@@ -45,9 +45,9 @@ In this tutorial, you will:
 
    ```typescript
    // app_test.ts
-
-   import { buildFor } from "./deps.ts";
-
+   
+   import { buildFor } from "https://deno.land/x/sinco@v4.1.0/mod.ts";
```

Comment:
ideally would still pont to `deps.ts` so we can reference the sinco version once at the start of the tutorial to help with auto bumping the version